castanet/counter.py

```diff
@@ -2,34 +2,28 @@
 from typing import List, Tuple, Dict, Optional
 # Import LibCST
 import libcst as cst
-import generate_trees as generator
-
+from castanet import generate_trees as generator
 import libcst.matchers as match
-    
-# TODO: Create function definitions related to all necessary user stories (should look somewhat like match_imports)
-# TODO: Modify annotations to have node key as type and key as necessary information
-
 
-def match_imports(cast_dict):
+def match_ifstatements(cast_dict):
```

Comment:
Similar to my below comment, please properly name this function in camel case: `match_if_statements`